### DISCUSSION TITLE
Correction required in 1.4.3 Plotting results - Bode Plot documentation.

### DISCUSSION THREAD

**Discussion by shishirsanjeeva** [petrobras/ross#1035] [Q&A]
Calculation error in 1.4.3 Plotting results - Bode Plot.
DoF=26*4+1=105

![Screenshot 2024-03-17 184825](https://github.com/petrobras/ross/assets/67865221/f1b1dc55-56a8-4402-9206-c0ee2a7725ee)

**Comment by jguarato** [comment]
Hi @shishirsanjeeva!

Sorry for the delay in responding and thank you for reporting this error. This has been addressed and it will be available soon in a fixed version of ROSS.

For future errors/bugs and issues you encounter in ROSS, we recommend reporting them as a new [Issue](https://github.com/petrobras/ross/issues), instead of addressing them here in the discussion area.

Thanks again.
